tests/lib/rules/no-restricted-properties.js

```diff
@@ -546,6 +546,138 @@ ruleTester.run("no-restricted-properties", rule, {
                 },
                 type: "MemberExpression"
             }]
+        }, {
+            code: "const { bar: { bad } = {} } = foo;",
+            options: [{ property: "bad" }],
+            parserOptions: { ecmaVersion: 6 },
```

Comment:
```suggestion
            languageOptions: { ecmaVersion: 6 },
```

Reply:
I did, ig the ci will fail since the base is an older commit, anything that I should do to fix it ?

Reply:
CI checks out from a merge commit, so the tests should pass.

Problem is only linting, as `languageOptions` should be at the same place where `parserOptions` were. If you could update that, it would be great.

Reply:
> Problem is only linting, as `languageOptions` should be at the same place where `parserOptions` were. If you could update that, it would be great.

```
2023-12-14T12:03:14.9805284Z /home/runner/work/eslint/eslint/tests/lib/rules/no-restricted-properties.js
2023-12-14T12:03:14.9909087Z ##[error]  559:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9922248Z ##[error]  570:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9927445Z ##[error]  581:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9932354Z ##[error]  592:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9937079Z ##[error]  603:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9941897Z ##[error]  614:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9946759Z ##[error]  625:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9952174Z ##[error]  636:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9957490Z ##[error]  647:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9962312Z ##[error]  658:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9967210Z ##[error]  669:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
2023-12-14T12:03:14.9972095Z ##[error]  680:13  error  The properties of a test case should be placed in a consistent order: [code, options, languageOptions, errors]  eslint-plugin/test-case-property-ordering
```

Reply:
Ye, done.